include/circt/Dialect/Calyx/CalyxControl.td

```diff
@@ -372,3 +372,38 @@ def StaticRepeatOp : CalyxContainer<"static_repeat", [
   ];
 
 }
+
+def RepeatOp : CalyxContainer<"repeat", [
+    ControlLike
+  ]> {
+  let summary = "Calyx Dynamic Repeat";
+  let arguments = (ins
+    I32Attr:$count
+  );
+  let description = [{
+    The "calyx.repeat" operation represents the repeated execution of
+    the control within its region.
+
+    ```mlir
+      calyx.repeat 10 {
```

Comment:
Isn't this a static repeat operation in the example? Maybe use a variable value instead of `10`?